tsl/src/fdw/modify_plan.c

```diff
@@ -54,21 +54,21 @@ get_update_attrs(RangeTblEntry *rte)
 static List *
 get_chunk_data_nodes(Oid relid)
 {
-	Chunk *chunk = ts_chunk_get_by_relid(relid, false);
+	Oid chunk_id = ts_chunk_get_id_by_relid(relid);
```

Comment:
Same here... chunk_id is int32 and not Oid (I know under the hood can be the same but we abstract those primitive types)